src/lib/Scenes/Artwork/Components/OtherWorks/ContextGridCTA.tsx

```diff
@@ -0,0 +1,26 @@
+import { ArrowRightIcon, Sans } from "@artsy/palette"
+import SwitchBoard from "lib/NativeModules/SwitchBoard"
+import React from "react"
+import { Text, TouchableWithoutFeedback } from "react-native"
+
+interface ContextGridCTAProps {
+  href?: string
+  label: string
+}
+
+export class ContextGridCTA extends React.Component<ContextGridCTAProps> {
+  render() {
+    const { href, label } = this.props
+
+    return (
+      <TouchableWithoutFeedback onPress={() => href && SwitchBoard.presentNavigationViewController(this, href)}>
+        <Text>
+          <Sans size="2" textAlign="left" weight="medium">
+            {label}
+          </Sans>
+          <ArrowRightIcon fill="black30" height="8px" />
```

Comment:
:thinking: should we either make href required or make this disappear when `href` is not given?

Reply:
Ooh good catch.

Reply:
Also, `8px`/`black30` seemed small/faint to me so I tried to find this on zeplin but couldn't find it on the responsive artwork specs or in the purchase ios artwork specs. Do you have a link?

Reply:
the icon in the spec is slightly different but I think making the text size `"3"`, leaving the icon at its default height, and adding a `ml={1}` on the icon achieves about the right weight balance.

```tsx
          <Sans size="3" textAlign="left" weight="medium">
            {label}
          </Sans>
          <ArrowRightIcon fill="black30" ml={1} />
```

![image](https://user-images.githubusercontent.com/1242537/60903817-ed6b2a80-a269-11e9-9e06-aefff4f99606.png)

Looks much better in context! :)

Reply:
This does look good! I was following the specs that @samchieng gave me in slack, but this does look a bit clearer.

Will see what she things! 🙏

Reply:
ooo yes this is much better, thanks @ds300 and @sweir27!